app/views/tweets/index.erb

```diff
@@ -0,0 +1,12 @@
+<h3>Welcome, to Fwitter <%=current_user.username%>!</h3>
+
+<% Tweet.all.each do |tweet| %>
```

Comment:
you should use that `@tweets` from the application controller I told you about here. Abstract the business logic from the view whenever possible.